components/ReadMore/ReadMore.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+export default function ReadMoreReadLess({ description }) {
+  const [isReadMore, setIsReadMore] = useState(true);
+  const toggleReadMore = () => {
+    setIsReadMore(!isReadMore);
+  };
+  return (
+    <StyledParagraph>
+      {isReadMore ? description.slice(0, 350) : description}
```

Comment:
Similar to above, it would be better to extract this number into a constant to make the code more maintainable and improve readability.